idl/chromadb/proto/chroma.proto

```diff
@@ -53,7 +53,7 @@ message Collection {
   optional int32 dimension = 5;
   string tenant = 6;
   string database = 7;
-  int64 logPosition = 8;
+  int64 log_position = 8;
```

Comment:
this should be snake_case - https://protobuf.dev/programming-guides/style/#message_and_field_names

Reply:
LGTM